src/main/java/data/DeckPathCleaner.java

```diff
@@ -0,0 +1,19 @@
+package data;
+
+public class DeckPathCleaner {
+    /**
+     * Removes path and extension from a file
+     * @param filePath where the file is located
+     * @return clean name of the file
+     */
+    public static String cleanName(String filePath){
```

Comment:
Much needed function. Only thing I could see going wrong is if people put '\' into their deck name. However, I think that we can implement these changes. Please include in the ReadMe the limitations on deckNames

Reply:
I'll can make it so it removes any quotation marks in the name while importing

Reply:
Actually nevermind I think I know how to include quotations without messing anything

Reply:
![image](https://user-images.githubusercontent.com/59342956/205468023-bdca07ba-b256-42d2-9beb-5f4834fd9bb6.png)
actually no change needed xD

Reply:
Perfect